lms/templates/course_modes/choose.html

```diff
@@ -12,6 +12,258 @@
     ${_("Enroll In {course_name} | Choose Your Track").format(course_name=course_name)}
```

Comment:
We caught up about this offline, but just to capture it here: you should be editing the edx.org theme for this page, not the base template.